web/packages/teleterm/src/sharedProcess/api/protogen/ptyHostService_grpc_pb.d.ts

```diff
@@ -1,19 +1,3 @@
-/**
- * Copyright 2023 Gravitational, Inc
```

Comment:
Those "hand-generated" files in this dir are weird, you need to run `make fix-license` now to add the license back.

I was supposed to move these files to the proto dir but I've never had the time to do this.